src/main/java/com/mitchej123/hodgepodge/core/HodgepodgeCore.java

```diff
@@ -28,34 +30,37 @@
 import com.mitchej123.hodgepodge.util.VoxelMapCacheMover;
 import com.mitchej123.hodgepodge.util.WorldDataSaver;
 
+import cpw.mods.fml.relauncher.FMLRelaunchLog;
 import cpw.mods.fml.relauncher.IFMLLoadingPlugin;
 
 @IFMLLoadingPlugin.MCVersion("1.7.10")
 @IFMLLoadingPlugin.TransformerExclusions({ "com.mitchej123.hodgepodge.asm", "optifine" })
-@IFMLLoadingPlugin.DependsOn("cofh.asm.LoadingPlugin")
 public class HodgepodgeCore implements IFMLLoadingPlugin, IEarlyMixinLoader {
 
-    private static File mcLocation;
     private static boolean isObf;
     private String[] transformerClasses;
 
     static {
+        EarlyConfig.ensureLoaded();
+        if (!EarlyConfig.noNukeBaseMod) {
+            String transformer = "com.mitchej123.hodgepodge.asm.transformers.early.ModContainerFactoryTransformer";
+            FMLRelaunchLog.finer("Registering transformer %s", transformer);
+            Launch.classLoader.registerTransformer(transformer);
```

Comment:
Another option would be what Angelica does:
1. If RFB is present, use a RFB plugin, mark that, and hack away
2. If RFB is not present, use this approach.

Reply:
why rely rfb if this works perfectly fine? it's not really a "hack" tbf that's just using the public api

Reply:
Moving towards a world in which RFB is required :angel: